### DISCUSSION TITLE
Min size of the RamDisk for Hardware Acceleration

### DISCUSSION THREAD

**Discussion by lzt007** [jellyfin/jellyfin#9840] [Q&A]
I'm trying to use a Ramdisk as cache disk for hardware acceleration.And i'm not sure whether 16GB is enough for this.

**Comment by Shadowghost** [comment]
This depends on the amount of transcoding happening simultaneously and the size/quality of the source content.

**Reply by lzt007** [reply]
Will the old cache be automatically cleaned up when the storage capacity is insufficient？

**Reply by Shadowghost** [reply]
No

**Reply by lzt007** [reply]
So if I play a 8GB vedio for one client in an uncompressed format，the server may need more than 8GB disk space to cache it. And that means it‘s better to ensure the cache space is larger than every single vedio on the server at least？And if the server needs to load multiple clients simultaneously，several times the size of a single video is needed due to the number of clients？
